### PR TITLE
Fix error when adding interesting tag

### PR DESCRIPTION
**Pull request by ZuluPro**
Instead of use `unicodedata.normalize`, I use `askbot.utils.slug.slugify`.
This should fix issue #606

### CODE REVIEW

File: askbot/forms.py [hunk 1]
```diff
@@ -427,7 +427,7 @@ def __init__(self, *args, **kwargs):
 
 def clean_tag(tag_name, look_in_db=True):
     """a function that cleans a single tag name"""
-    tag_name = unicodedata.normalize('NFC', tag_name)
+    tag_name = slugify(tag_name)
```
Comment (evgenyfadeev) [inline askbot/forms.py]: this I think won't work. Normalization was added to work with compound (e.g. accented) characters.

**Comment by evgenyfadeev** [comment]
ended up decoding the request.body where it is used.

**Comment by ZuluPro** [comment]
@evgenyfadeev 
Sorry, I should give more informations, below what I understood and did:

- I'm using askbot configured in French
- Tag's name can contains only ASCII characters, if I try to add a new one (from retag of question) it will slugify it, for example: € => eur, àpropos => apropos, élégance => elegance
- So when we search for tags, to have the same method of slugification could be better.
- With my patch, filters are saved slugified with the "same method" that the tags 
(There's a little display bug: JavaScript don't show them slugified, but they are save as)

**Comment by evgenyfadeev** [comment]
@ZuluPro tags are actually not limited to ascii chars, so that's not an issue - unless you insist that you actually want to limit tags to ASCII for some reason - in that case Askbot is not doing what you want.

There was a technical issue of incorrectly handing the unicode. In the past versions of django django.utils.simplejson.loads would automatically decode the input string (I found that yesterday after testing all currently supported versions of askbot 0.7.x branch and other branches 0.8.x 0.9.x and master, 0.7.x worked fine with django 1.5, others displayed issue #606) request.body comes as encoded string and loads from posted json returns and passes downstream data of type str, not unicode and the unicodedata.normalize function failed. The solution was to decode the request.body before running through json.loads

**Pull request closed without merging**